Firebase/Auth/Source/AuthProvider/OAuth/FIROAuthProvider.m

```diff
@@ -169,6 +169,9 @@ - (nullable instancetype)initWithProviderID:(NSString *)providerID auth:(FIRAuth
   NSAssert(![providerID isEqual:FIRFacebookAuthProviderID],
            @"Sign in with Facebook is not supported via generic IDP; the Facebook TOS "
            "dictate that you must use the Facebook iOS SDK for Facebook login.");
+  NSAssert(![providerID isEqual:@"apple.com"],
+           @"Sign in with Apple is not supported via generic IDP; You must use the Apple iOS SDK"
+           " for sign in with Apple.");
```

Comment:
Nit: Capital `S` for `Sign in with Apple` as per https://developer.apple.com/sign-in-with-apple/